docs/userGuide/syntax/popovers.md

```diff
@@ -71,6 +71,8 @@
 </variable>
 </include>
 
+<include src="includes.md#baseUrl-warning"/>
```

Comment:
<img width="952" alt="Screenshot 2024-04-12 at 8 34 22 PM" src="https://github.com/MarkBind/markbind/assets/59471320/230551d9-1a12-4832-8ab7-904472fcd4f6">
When embedded, it seems like the warning is very large and disrupts the flow of the text. Actually, even when it's not included the placing feels a little awkward. What if all or some of the content was in an expandable panel, so that it's out of the way? Just the header might be good? I suppose you could also experiment with having the example be expandable instead.

Reply:
yep sure, I've put the example into a panel and changed the explanation to make it more clear and generic!